amnesia/src/game/LuxMapHandler.cpp

```diff
@@ -395,7 +397,7 @@ void cLuxMapHandler::ChangeMap(const tString &asMapName, const tString &asStartP
 cLuxMap *cLuxMapHandler::LoadMap(const tString &asFileName, bool abLoadEntities) {
 	const tString & mapname = FileToMapName(asFileName);
 
-	gpBase->mpEffectHandler->GetColorGrading()->InitializeLUT(mapname + "_colorgrading.png");
+	//gpBase->mpEffectHandler->GetColorGrading()->InitializeLUT(mapname + "_colorgrading.png");
```

Comment:
Why is this commented out instead of using a preprocessor check?

Reply:
Commented out for the aforementioned reasons.

![Screenshot_20240705_054432_Discord.png](https://github.com/TiManGames/AmnesiaTheDarkDescent/assets/82419801/54d3c808-0ea7-4d8c-89cd-caf3e7b165a8)

Reply:
The bottom line is that *theoretically* the color grading areas should work without this.